build/common/installer/scripts/tomlparser.rb

```diff
@@ -25,6 +25,9 @@
 @adxDatabaseName = "containerinsights" # default for all configurations
 @logEnableMultiline = "false"
 @stacktraceLanguages = "go,java,python,dotnet"
+@logEnableKubernetesMetadata = false
+@logKubernetesMetadataIncludeFields = "podLabels,podAnnotations,podUid,image"
```

Comment:
As we discussed during bug bash, we would need to address items which we have identified during bug bash as part of the public preview.

Reply:
seperated image into four fields but configmap for each field is pending implementation

Reply:
added